src/client/containers/SignUp/SignUp.css

```diff
@@ -0,0 +1,27 @@
+.sign-up-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.sign-up-left {
+  flex: 1 1 auto;
+}
+
+.sign-up-right {
+  flex: 1 1 auto;
+}
+
+nav {
+  display: none;
+}
+
+div.footer {
+  display: none;
+}
+
```

Comment:
No global styles, scope them with a class name

Reply:
@bytewiz would you suggest that I give the nav and the footer classes on App.js? I know it is not very good to use the global styles, but as the css is just being imported for the index of the Sign-up page, can it have a negative impact somewhere else?